lib/matplotlib/pyplot.py

```diff
@@ -2486,7 +2481,7 @@ def polar(*args, **kwargs) -> list[Line2D]:
 # is compatible with the current running interactive framework.
 if (rcParams["backend_fallback"]
         and rcParams._get_backend_or_none() in (  # type: ignore
-            set(rcsetup.interactive_bk) - {'WebAgg', 'nbAgg'})
+            set(backendRegistry.list_builtin(BackendFilter.INTERACTIVE_NON_WEB)))
```

Comment:
```suggestion
            backendRegistry.list_builtin(BackendFilter.INTERACTIVE_NON_WEB))
```
The set is not needed anymore. it was only used to exclude WebAgg and nbAgg.

Reply:
Good catch.